go.mod

```diff
@@ -48,15 +48,15 @@ require (
 	github.com/pingcap/check v0.0.0-20211026125417-57bd13f7b5f0
 	github.com/pingcap/errors v0.11.5-0.20211224045212-9687c2b0f87c
 	github.com/pingcap/failpoint v0.0.0-20220303073211-00fea37feb66
-	github.com/pingcap/kvproto v0.0.0-20220314103629-10e688307221
+	github.com/pingcap/kvproto v0.0.0-20220328072018-6e75c12dbd73
 	github.com/pingcap/log v0.0.0-20211215031037-e024ba4eb0ee
-	github.com/pingcap/tidb v1.1.0-beta.0.20220318092833-a30449081049
+	github.com/pingcap/tidb v1.1.0-beta.0.20220329110428-fbaaa1117936
 	github.com/pingcap/tidb-tools v6.0.0-alpha.0.20220317013353-dfc5146f4746+incompatible
```

Comment:
Why is it still here?

Reply:
in the tidb-tools, some libraries import tiflow. so it needs to update the tiflow and then migrate this library into tidb. They are waiting for this PR.